docs/project/2024/ana.teodorescu2203/index.md

```diff
@@ -0,0 +1,119 @@
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Raspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [GA12-N20](https://www.handsontec.com/dataspecs/GA12-N20.pdf) | Micro Motor | [26 RON](https://www.optimusdigital.ro/ro/motoare-micro-motoare-cu-reductor/681-micro-motor-cu-reductor-ga12-n20-1200.html?search_query=Micro+Motor+GA12-N20+cu+Reductor+1%3A200+%C8%99i+Ax+de+10+mm&results=7) |
+| 2x JA12-N20 | Micro Motor | [90 RON](https://www.optimusdigital.ro/ro/motoare-micro-motoare-cu-reductor/237-motor-cu-reductor-ja12-n20-.html?search_query=Micro+Motor+cu+Reductor+JA12-N20+1%3A298&results=2) |
+| [Dual Motor Driver L9110s](https://www.laskakit.cz/user/related_files/l9110_2_channel_motor_driver.pdf) | 2x Motor driver | [8 RON](https://www.optimusdigital.ro/ro/drivere-de-motoare-cu-perii/480-driver-de-motoare-l9110s.html?search_query=Driver+de+Motoare+Dual+L9110S&results=2) |
+| [2x U Shape Photoelectric Sensor](https://datasheet.octopart.com/PM-L54-Panasonic-datasheet-33007024.pdf)| Sensors | [14 RON](https://www.optimusdigital.ro/en/optical-sensors/599-senzor-fotoelectric-in-miniatura-in-forma-de-u.html) |
+| [Active Buzzer](https://components101.com/misc/buzzer-pinout-working-datasheet) | Active Buzzer | [4 RON](https://www.optimusdigital.ro/ro/audio-buzzere/8177-buzzer-activ-sfm-20b-3-24-v.html?search_query=Buzzer+Activ+SFM-20B+%283+-+24+V%29&results=1) |
+| 10x Pulley Wheel | Pulley Wheel | [10 RON](https://www.optimusdigital.ro/ro/mecanica-roti-scripete/406-roata-scripete-de-93-mm.html?search_query=Roata+Scripete+Alba+%CE%A69.3-2A&results=1) |
+| 4x Switch | Switch | [4 RON](https://www.optimusdigital.ro/ro/butoane-i-comutatoare/1120-comutator-in-miniatura.html?search_query=Comutator+in+Miniatura&results=1) |
+| Battery Holder 4xR6 | Battery Holder | [9 RON](https://www.optimusdigital.ro/ro/suporturi-de-baterii/2806-suport-baterii-4-x-r6.html?search_query=Suport+baterii+4+x+R6&results=63) |
+| [Male to male jumper wires](https://media.digikey.com/pdf/Data%20Sheets/Digi-Key%20PDFs/Jumper_Wire_Kits.pdf) | Wires | [7 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/884-set-fire-tata-tata-40p-10-cm.html?search_query=Fire+Colorate+Tata-Tata+%2840p%2C+10+cm%29&results=10) |
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [pwm](https://docs.rs/rp2040-hal/latest/rp2040_hal/pwm/index.html) | Pulse-width modulation | Used for controlling the buzzer |
+| [gpio](https://docs.embassy.dev/embassy-stm32/git/stm32c011d6/gpio/index.html) | GPIO manipulation | Used for interacting with GPIO pins |
```

Comment:
This is for STM32, not Raspberry Pi.

Reply:
I also deleted this because the support for the GPIO pins is already included in the embedded-hal library